bff/src/Altinn.Authentication.UI/Altinn.Authentication.UI.Core/SystemUsers/ISystemUserService.cs

```diff
@@ -14,7 +14,7 @@ public interface ISystemUserService
     /// <summary>
     /// Creates a system user
     /// </summary>
-    Task<SystemUser?> CreateSystemUser(int partyId, SystemUserDescriptor newSystemUserDescriptor, CancellationToken cancellation = default);
+    Task<(SystemUser?, string?)> CreateSystemUser(int partyId, SystemUserDescriptor newSystemUserDescriptor, CancellationToken cancellation = default);
```

Comment:
Please don't use a tuple as a result type. This is not go. If you want to return a Result type, then make a result type that makes it clear from the type signature what's actually going on here. Alternatively (and probably preferably) use https://github.com/Altinn/altinn-authorization-utils/blob/main/src/Altinn.Authorization.ProblemDetails/src/ProblemDetails.Abstractions/Result.cs from the `Altinn.Authorization.ProblemDetails.Abstractions` package.